session.go

```diff
@@ -151,21 +151,26 @@ func (s *Session) Get(ctx *fasthttp.RequestCtx) (Storer, error) {
 		return nil, errNotSetProvider
 	}
 
+	newSessionGenerated := false
```

Comment:
This is unnecessary, revert it please.

Reply:
var newSessionGenerated bool instead?

Reply:
I mean to not allocate a new variable to check if new session is generated. Put directly `s.setHTTPValues(ctx, sessionID)` inside condition, like this: https://github.com/fasthttp/session/blob/master/session.go#L161

Reply:
I need the expiration since the signature is now `setHTTPValues(ctx *fasthttp.RequestCtx, sessionID []byte, expires time.Duration)` and I need the store to get the expiration value. 

Otherwise I can revert the change and use the configured expiration from the global config but it's a weak design since now I think it's the responsibility of the store to decide whether it wants to consume the configured expiration or not.

I will propose something else to avoid the allocation but still get the expiration from the store.

Reply:
Well, actually the other solution was to declare the store before the if block and do an if/else and assign store in both blocks but this solution also pre-allocates the store variable anyway. I think the current solution is the best we can get from a architecture perspective but if you want I can avoid the allocation by setting the expiration coming from the config, it just creates a small weakness IMHO. I let you tell me.

Reply:
Sorry, I don't see you get expiration from store.

Reply:
look up to line 172, you'll understand what I mean.